### DISCUSSION TITLE
It is recommended to add a right-to-left reading mode

### DISCUSSION THREAD

**Discussion by shenshui** [sumatrapdfreader/sumatrapdf#3831] [Ideas]
The recommended addition of reading order is right-to-left double-page reading mode and book reading mode.

That is, page 1 is displayed on the right and page 2 is displayed on the left.

Vertical lines of text need this reading mode to read properly.

**Comment by GitHubRulesOK** [comment]
see https://github.com/sumatrapdfreader/sumatrapdf/issues/1264#issuecomment-1150241521
